### DISCUSSION TITLE
flytectl demo fails to start

### DISCUSSION THREAD

**Discussion by ALBDN** [flyteorg/flyte#5378] [Q&A] [answered]
Hi,

I am trying to run a project locally following the tutorial _Running a workflow in a local Flyte cluster_
However when running _flytectl demo start_ I get the following error:
`🧑‍🏭 Bootstrapping a brand new Flyte cluster... 🔨 🔧
Error: Malformed version: flytectl/v0.8.20
{"json":{},"level":"error","msg":"Malformed version: flytectl/v0.8.20","ts":"2024-05-16T15:27:30+02:00"}`

Any idea about what could be wrong ?
It is run on an Ubuntu 22.04.
Any help would be greatly appreciated!

Thank you!

**Comment by davidmirror-ops** [comment]
Hi @ALBDN thanks for reporting.

flytectl was recently migrated to the monorepo, which is good but caused some temporary hiccups.

Please use your package manager to upgrade flytectl and it should fix any issues.

Example:
```
brew upgrade flytectl
Upgrading flyteorg/tap/flytectl
  0.8.15 -> 0.8.20
```

**Reply by ALBDN** [reply]
Thank you for the reply @davidmirror-ops!

I understand from your reply above that I should have the 0.8.20 version.
The thing is doing a:
_flytectl upgrade_ 
returns: _You already have the latest version of Flytectl_ (which is 0.8.18) 
Am I missing something about the upgrade ?

**Reply by davidmirror-ops** [reply]
We're investigating this. In the meantime could you try with `flytectl demo start --version 1.12.0`?

**Comment by ALBDN** [comment]
Hi,
Just tried _flytectl demo start --version 1.12.0_ but failing with the following error:
`Error: GET https://api.github.com/repos/flyteorg/flyte/releases/tags/1.12.0: 404 Not Found []
{"json":{},"level":"error","msg":"GET https://api.github.com/repos/flyteorg/flyte/releases/tags/1.12.0: 404 Not Found []","ts":"2024-05-17T09:51:49+02:00"}`

**Reply by davidmirror-ops** [reply, marked as answer]
So sorry, it was supposed to be:

`flytectl demo start  --version v1.12.0`

**Reply by ALBDN** [reply]
Thank you @davidmirror-ops! It's working!
